CHANGELOG.md

```diff
@@ -6,6 +6,230 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+  (old names: `Interpreter.[METHOD_NAME]()`),
+  PR [#499](https://github.com/ethereumjs/ethereumjs-vm/pull/499)
+- Further simplification of `Interpreter`/`EVM`
+  (old: `Loop`/`Interpreter`) structure,
+  PR [#506](https://github.com/ethereumjs/ethereumjs-vm/pull/506)
+- [BREAKING] Dropped `VM.runJit()` in favor of direct handling in
+  `EVM` (old: `Interpreter`),
+  officially not part of the external API but mentioning just in case,
+  PR [#515](https://github.com/ethereumjs/ethereumjs-vm/pull/515)
+- Removed `StorageReader`, moved logic to `StateManager`,
+  [#534](https://github.com/ethereumjs/ethereumjs-vm/pull/534)
+
+### Istanbul Process Start
+
+With this release we start the `Istanbul` hardfork integration process and
+have activated the `istanbul` `hardfork` option for the constructor.
+
+This is meant to be used experimentation and reference implementations, we have made
+a start with integrating draft [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108)
+`Istanbul` candidate support reducing the gas costs for `alt_bn128` precompiles,
+see PR [#539](https://github.com/ethereumjs/ethereumjs-vm/issues/539) for
+implementation details.
+
+Note that this is still very early in the process since no EIP in a final
+state is actually accepted for being included into `Istanbul` on the time of
+release. The `v4` release series will be kept as an experimental series
+during the process with breaking changes introduced along the way without too
+much notice, so be careful and tighten the VM dependency if you want to give
+your users the chance for some early experimentation with some specific
+implementation state.
+
+Once scope of `Istanbul` as well as associated EIPs are finalized a stable
+`Istanbul` VM version will be released as a subsequent major release.
+
+### Code Modernization and Version Updates
+
+The main API with the `v4` release switches from being `callback` based to
+using promises,
+see PR [#546](https://github.com/ethereumjs/ethereumjs-vm/pull/546).
+
+Here is an example for changed API call `runTx`.
+
+Old `callback`-style invocation:
+
+```javascript
+vm.runTx(
+  {
+    tx: tx,
+  },
+  function(err, result) {
+    if (err) {
+      // Handle errors appropriately
+    }
+    // Do something with the result
+  },
+)
+```
+
+Promisified usage:
+
+```javascript
+try {
+  let result = await vm.runTx({ tx: tx })
+  // Do something with the result
+} catch (err) {
+  // handle errors appropriately
+}
+```
+
+##### Code Modernization Changes
+
+- Promisified internal usage of async opcode handlers,
+  PR [#491](https://github.com/ethereumjs/ethereumjs-vm/pull/491)
+- Promisified `runTx` internals,
+  PR [#493](https://github.com/ethereumjs/ethereumjs-vm/pull/493)
+- Promisified `runBlock` internals, restructure, reduced shared global state,
+  PR [#494](https://github.com/ethereumjs/ethereumjs-vm/pull/494)
+
+##### Version Updates
+
+- [BREAKING] Updated `ethereumjs-account` from `2.x` to `3.x`, part of
```

Comment:
Not sure I get why this is a breaking change?

Reply:
Was meant in the sense of "people have to update their dependencies", but you are right, not really something which can be considered "breaking", have removed the label.

Reply:
Once you see and are fine with the update please re-approve.